src/globalUiComponents.js

```diff
@@ -0,0 +1,8 @@
+import Vue from "vue"
+
+Vue.component("ui-debio-card", () => import("@/core/components/ui/Card"))
```

Comment:
Please move this inside the `common\components\` directory. You can name this file `index.js` or `global.js` if you prefer.